luminaire/tests/conftest.py

```diff
@@ -69,6 +69,21 @@ def training_test_data():
 
     return data
 
+@pytest.fixture(scope='session')
+def training_test_data_zeroes():
+    """
+    Data with all zeroes to test lad structural training
+    """
+
+    data = pd.read_csv(get_data_path('daily_test_time_series.csv'))
+    data['index'] = pd.DatetimeIndex(data['index'])
+    data['raw'] = np.zeros_like(np.array(data['raw']))
```

Comment:
not sure if `data['raw'].apply(lambda _: 0.0)` will be better here

Reply:
As far as I can see these all produce identical results, so I'd prefer the code you already have since it's the easiest to read and most performant of them all. `apply` requires running code in python as opposed to lower level C; `replace` does an unnecessary regex match.